inmoose/diffexp/meta.py

```diff
@@ -99,10 +99,17 @@ def meta_de(de_results, alpha=0.05, min_common_genes=None):
 
     # combining p-values
     pvals = {g: [d.loc[g, "pvalue"] for d in de_results] for g in idx}
-    meta_pvals = np.array([combine_pvalues(pvals[g]).pvalue for g in idx])
+    meta_pvals = np.array(
```

Comment:
I don't understand how you can have nans here based on what is done before?
Either all meta analyses have the same set of genes, or you take the intersection no?

Reply:
In RNA-Seq, if a gene has 0 count in all samples, then DESeq2/edgepy are not able to fit a negative binomial distribution on it, so they return NaN for the logFC, p-value... for this gene.
Therefore, it can happen that the gene is present in the datasets on which the meta-analysis is done, but the associated p-value is nan in one or several of them.

Reply:
Ok thanks!